R/slide.R

```diff
@@ -19,24 +20,32 @@
 #' @param ... Additional arguments to pass to the function or formula specified
 #'   via `f`. Alternatively, if `f` is missing, then the current argument is
 #'   interpreted as an expression for tidy evaluation. See details.  
-#' @param n Number of time steps to use in the running window. For example, if
-#'   `n = 7`, one time step is one day, and the alignment is "right", then to
-#'   produce a value on January 7 we apply the given function or formula to data
-#'   in between January 1 and 7.
+#' @param before A nonnegative integer specifying the number of time steps
+#'   before the `ref_time_value` to use in the running window.
+#'   This must be a vector of length 1.
+#'   Set to 0 for a right-aligned/trailing sliding window, meaning that no
```

Comment:
Somehow, this doesn't sound correct to me. If before=0 and after=5, then this is left-aligned and leading. Right?

Reply:
Think you're right, this should be left-aligned/leading, and in `@param after` setting after=0 is right-aligned/trailing.

Separately: explicitly passing 0 isn't even required; maybe this should be "Set to 0 or leave unset".

Reply:
You're right. I got them mixed up.